server/protocols/jmap-draft-integration-testing/jmap-draft-integration-testing-common/src/test/resources/cucumber/GetMessages.feature

```diff
@@ -447,6 +447,6 @@ Feature: GetMessages method
     And the first attachment is:
     |key      | value                        |
     |type     |"text/calendar"               |
-    |size     |1056                          |
+    |size     |1096                          |
```

Comment:
because of the encoding change in SMTPMessageSender (ISO-8859-1 to UTF-8). UTF-8 requires more bytes to encode some characters.